From e41f7b7b194dc960129e76f17717fd22c17df709 Mon Sep 17 00:00:00 2001
From: Hugh Dickins <hughd@google.com>
Date: Sun, 3 Sep 2017 19:51:10 -0700
Subject: [PATCH 34/43] kaiser: align addition to x86/mm/Makefile

Use tab not space so they line up properly, kaslr.o also.

(cherry picked from Change-Id: I2f2b64c130b1bafe7eceb5b2a3d5be9a9014b02b)
Conflicts:
arch/x86/mm/Makefile
---
 arch/x86/mm/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/x86/mm/Makefile
+++ b/arch/x86/mm/Makefile
@@ -32,4 +32,4 @@ obj-$(CONFIG_ACPI_NUMA)		+= srat.o
 obj-$(CONFIG_NUMA_EMU)		+= numa_emulation.o
 
 obj-$(CONFIG_X86_INTEL_MPX)	+= mpx.o
-obj-$(CONFIG_KAISER) += kaiser.o
+obj-$(CONFIG_KAISER)		+= kaiser.o
